cypress/integration/org_wallet.spec.js

```diff
@@ -57,6 +57,22 @@ context("navigation", () => {
       cy.pick("amount-input").type("123");
       cy.pick("send-transaction-button").click();
     });
+    it("tests the button is disabled when there is no amount is filled in", () => {
+      cy.pick("send-tab").click();
+      cy.pick("send").should("exist");
+      cy.pick("recipient-input").should("exist");
+      cy.pick("recipient-input").type(
+        "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+      );
```

Comment:
I think it is not necessary to check if the input exists and then enter the input. If you enter it directly, the test will fail anyway because it can't find the data-cy.
(Applies to the other tests added here as well)

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/712/commits/0f58c678731a91801d6d0bc6f01096cc72045ef3